ts/sagas/wallet/pagopaApis.ts

```diff
@@ -814,13 +820,18 @@ export function* paymentIdPollingRequestHandler(
       yield* select(isPagoPATestEnabledSelector);
 
     const getPaymentId = getPaymentIdApi.e2;
-    const response: SagaCallReturnType<typeof getPaymentId> = yield* call(
-      getPaymentId,
-      {
-        codiceContestoPagamento: action.payload.codiceContestoPagamento,
-        test: isPagoPATestEnabled
-      }
-    );
+
+    const request = getPaymentId({
+      codiceContestoPagamento: action.payload.codiceContestoPagamento,
+      test: isPagoPATestEnabled
+    });
+
+    const response: SagaCallReturnType<typeof getPaymentId> = (yield* call(
+      withRefreshApiCall,
+      request,
+      action
+    )) as unknown as SagaCallReturnType<typeof getPaymentId>;
```

Comment:
```suggestion
    )) as SagaCallReturnType<typeof getPaymentId>;
```
On my end `TSC` does not complain, if it doesn't on your end either we can remove the double casting 👍